Core/Utilities/Time.py

```diff
@@ -236,6 +236,17 @@ def includes( self, myDateTime ):
       return False
     return True
 
+def queryTime(f):
```

Comment:
In this same module there's a "timeThis" function, also to be used as decorator, that is used in several parts of the TransformationSystem in LHCb. It looks very similar to this one. Can you please check if that can be used? In order to avoid duplications.